versionInfo.gradle

```diff
@@ -37,47 +37,34 @@ def gatlingVersion() {
 // don't include paren, and don't use the longer syntax like:
 //   compile group: 'com.h2database', name: 'h2', version: "${versions.h2}"
 project.ext.versions = [
-    asciiDoctor: '0.1.4'
-    , asm: '5.0.3'
-    , checkstyle: '8.11'
-    , commonsCodec: '1.6'
+    checkstyle: '8.11'
     , commonsConfiguration: '1.9'
     , commonsIo: '2.4'
-    , commonsLang3: '3.4'
     , coveralls: '2.8.2'
-    , dispatch: '0.13.0'
     , flyway: '5.1.1'
-    , freemarker: '2.3.19'
-    // gatling 2.2.5 uses scala 2.11.x, while 2.3.1 uses scala 2.12.x
     , gatling: gatlingVersion()
     , gradleGit: '1.7.2'
     , gradleScoverage: '2.3.0'
-    , groovyHttpBuilder: '0.7.2'
     , guava: '25.1-jre'
-    , guice: '4.1.0'
+    , guice: '4.2.2'
     , gson: '2.8.0'
     , h2: '1.4.191'
-    , hamcrest: '1.3'
     , influxDbClient: '2.14'
-    , jacksonDataBind: '2.7.4'
-    , jbake: '1.0.0'
-    , json4s: '3.4.2'
     , junit: '4.12'
     , math3: '3.5'
-    , mysqlConnector: '5.1.46'
+    , mysqlConnector: '5.1.47'
```

Comment:
is there anything preventing us from using mysqlJ 8.0?

Reply:
it didnt work haha. broke the code generator, i dug into it a bit, and somehow slick `MySQLProfile.defaultTables` returns an empty collection.

Reply:
oh right lol